packages/renderer/src/lib/volume/VolumeColumnEnvironment.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+import ProviderInfo from '../ui/ProviderInfo.svelte';
+import type { VolumeInfoUI } from './VolumeInfoUI';
+
+export let object: VolumeInfoUI;
```

Comment:
Same here, could we pass the useful info only (engineName, engineId), and have a generic and resusable ProviderColumn?

Reply:
Same as above.